pkg/cmd/server/api/validation/oauth.go

```diff
@@ -18,8 +18,16 @@ import (
 func ValidateOAuthConfig(config *api.OAuthConfig) ValidationResults {
 	validationResults := ValidationResults{}
 
+	if config.MasterCA != nil && len(*config.MasterCA) > 0 {
```

Comment:
moved the defaulting for this to a defaulting func on the internal version and removed the nil check